### DISCUSSION TITLE
Does Selenide support browserInDocker from WebDriverManager 5?

### DISCUSSION THREAD

**Discussion by bjelac** [selenide/selenide#2109] [General]
Does Selenide support browserInDocker from WebDriverManager 5? 
If it does, is there a code snippet that shows how to use it?
I have search for it on google but getting only Selenium results back.

**Comment by asolntsev** [comment]
Selenide supports everything that works in Selenium. 
Any example with Selenium can be easily converted to Selenide syntax.
